integration-cli/utils/utils_test.go

```diff
@@ -0,0 +1,490 @@
+// +build !windows
```

Comment:
I don't like this too much as it's gonna be skipped on Windows but.. :sweat_smile: